meson.build

```diff
@@ -127,8 +138,16 @@ endif
 subdir('va')
 subdir('pkgconfig')
 
-doxygen = find_program('doxygen', required: false)
+doxygen = find_program('doxygen', required : get_option('enable_docs'))
 
-if get_option('enable_docs') and doxygen.found()
+if get_option('enable_docs')
```

Comment:
what if enable_docs is auto but doxygen is not installed?

Reply:
This is currently broken and should be switched to use `if doxygen.found()`.

Reply:
The enable_docs can never be auto - it's a bool.

Reply:
> This is currently broken and should be switched to use `if doxygen.found()`.

```
$ meson -D enable_docs=true

Program doxygen found: NO

meson.build:141:0: ERROR: Program 'doxygen' not found or not executable
```

Seems to be working as expected. Note the option is a bool and not a tristate (combo or feature).